### DISCUSSION TITLE
Toggle collapse section

### DISCUSSION THREAD

**Discussion by appietr** [Runjuu/react-simple-resizer#5] [General]
I want to have a collapsible section with a minimum width. So the idea is that there is a toggle button that collapses/un-collapses the section, but it can never be adjusted below a specific width. 

So what I did is conditionally render the collapsible section with the && in React, but as soon as I toggle it again, the section pops back at its minimum size, but it is no longer adjustable...

Is there a better way to do it?

**Comment by runjuu** [comment]
It seems pretty similar to the [collapsible section example](https://column-resizer.runjuu.com/examples/collapsible-section), and you can find the implementation [here](https://github.com/runjuu/column-resizer/blob/main/docs/examples/collapsible-section.tsx).
